### DISCUSSION TITLE
Celery flower keeps restarting

### DISCUSSION THREAD

**Discussion by pungys97** [celery/celery#9229] [Q&A]
Ok so we have celery flower configured for monitoring. And it is working with some "buts". Generally there is a big delay between task submitted and shown in flower (we see the tasks with certain delay of up to few hours). I don't know if this is load/configuration/skill issue.

When I try to look into it the weirdest thing is that flower keeps restarting. When I run `supervisorctl status` I always get `RUNNING` for few seconds or `STARTING`. Is this expected or is this some issue worth solving?
<img width="587" alt="Screenshot 2024-09-03 at 11 09 25 AM" src="https://github.com/user-attachments/assets/23661a48-49a8-4a96-91cc-e7309aaa5fe3">

**Comment by techUdayMungalpara** [comment]
yo the flower restarts r def not normal! check the logs:

```
supervisorctl tail flower
```

whats ur broker?  might be choking on smth

let's fix that restart loop first tho - its probably crashing for a reason. dump those logs and we'll figure it out!
